desktop-app/app/components/Addressinput/index.js

```diff
@@ -175,6 +224,64 @@ class AddressBar extends React.Component<Props> {
     }
     return address;
   };
+
+  _addUrlToExistingSearchResult = () => {
```

Comment:
Can you please move this logic to the reducer? Under the `NEW_ADDRESS` reducer? 

It fits more there than handled here.